### PR TITLE
Translate attention.md into Chinese

### PR DESCRIPTION
**Pull request by wwwbai**
# What does this PR do?

Hello, I am a beginner of open source project. This PR translates attention.md into Chinese.Please let me know if there are any omissions, and I will be happy to revise them.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by stevhliu — COMMENTED** [review]
Thanks for your translation! Do you know someone who can read Chinese and would be interested in reviewing your translation? I can only review the high-level things because I can't read Chinese 😅

**Comment by wwwbai** [comment]
Oh, I'm sorry, I'm a first-time contributor to the open source community, so I may not know where to find someone who can read Chinese.Or can I find someone who can read Chinese under the comments of other issues that translate documents into Chinese?

> Thanks for your translation! Do you know someone who can read Chinese and would be interested in reviewing your translation? I can only review the high-level things because I can't read Chinese 😅

**Comment by wwwbai** [comment]
Maybe @Isotr0py would you like to preview my PR? I found you under other issues related to Chinese translation.

File: docs/source/zh/attention.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+除非适用法律要求或书面同意，否则按“按原样”分发的软件，无论是明示还是暗示的，都没有任何担保或条件。请参阅许可证以了解特定语言下的权限和限制。
+
+⚠️ 请注意，本文件虽然使用Markdown编写，但包含了特定的语法，适用于我们的doc-builder（类似于MDX），可能无法在您的Markdown查看器中正常渲染。
+
+-->
+
+# 注意力机制
+
+在注意力矩阵是正方形的意义上，大多数变压器模型使用完全注意力。当你有很长的文本时，这可能是一个很大的计算瓶颈。Longformer和reformer是试图提高效率的模型，它们使用注意力矩阵的稀疏版本来加速训练。
```
Comment (Isotr0py) [inline docs/source/zh/attention.md:15]:
```suggestion
大多数 transformer 模型使用完全注意力机制，该机制采用正方形的注意力矩阵。当输入很长的文本时，这将导致巨大的计算瓶颈。Longformer 和 Reformer 是提高注意力机制效率的改进模型，它们使用稀疏化的注意力矩阵来加速训练。
```

File: docs/source/zh/attention.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <img scale="50 %" align="center" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local_attention_mask.png"/>
+</div>
+
+使用那些具有较少参数的注意力矩阵允许模型具有更大序列长度的输入。
+
+## 其他技巧
+
+### 轴向位置编码
+
+[Reformer](model_doc/reformer)使用轴向位置编码:在传统的transformer模型中，位置编码E是一个大小为 \\(l\\) 乘 \\(d\\) 的矩阵，其中 \\(l\\) 是序列长度，而 \\(d\\) 是隐藏状态的维数。如果你有很长的文本，这个矩阵会很大，会占用GPU太多的空间。为了减轻这种情况，轴向位置编码包括将大矩阵E分解成两个更小的矩阵E1和E2，维数为 \\(l_{1} \乘以d_{1}\\) 和 \\(l_{2} \乘以 d_{2}\\),使得 \\(l_{1} \乘以 l_{2} = l\\) 和 \\(d_{1} + d_{2} = d\\) (对于长度的乘积，这最终要小得多)。E中时间步长 \\(j\\) 的嵌入是通过连接E1中时间步长 \\(j \% l1\\) 和E2中时间步长 \\(j // l1\\) 的嵌入而获得的。
```
Comment (Isotr0py) [inline docs/source/zh/attention.md:36]: Why `\times` command used in latex was translated to "\乘"?

**Review by Isotr0py — CHANGES_REQUESTED** [review]
To be honest, this translation is quite in a style of machine translation, with many terminologies and code structures translated incorrectly...

I think you have better take a look at the dictionary in Chinese README to see how to deal with terminology: https://github.com/huggingface/transformers/blob/13493215abceafc1653af88b045120014fb4c1fc/i18n/README_zh-hans.md?plain=1#L22-L39

For some terminologies not listed in README, you can refer to [Google Developers Documentation](https://developers.google.cn/machine-learning/glossary?hl=zh-cn#Transformer), they have provided a dictionary for machine learning terminology.

File: docs/source/zh/attention.md (hunk 1 above)
Comment (wwwbai) [reply docs/source/zh/attention.md:15]: Thank you for your advice! I'm sorry that I didn't fully understand this concept, so I translated it like this. I'll revise my translation right away.

File: docs/source/zh/attention.md (hunk 2 above)
Comment (wwwbai) [reply docs/source/zh/attention.md:36]: I'm so sorry I didn't realize it was a latex text at first. I really lack this knowledge. I'm going to study it carefully now.

**Comment by wwwbai** [comment]
I submitted a new PR(#34781 ), could you please take a look at it?Thank you very much!

> To be honest, this translation is quite in a style of machine translation, with many terminologies and code structures translated incorrectly...
> 
> I think you have better take a look at the dictionary in Chinese README to see how to deal with terminology:
> 
> https://github.com/huggingface/transformers/blob/13493215abceafc1653af88b045120014fb4c1fc/i18n/README_zh-hans.md?plain=1#L22-L39
> 
> For some terminologies not listed in README, you can refer to [Google Developers Documentation](https://developers.google.cn/machine-learning/glossary?hl=zh-cn#Transformer), they have provided a dictionary for machine learning terminology.

**Comment by stevhliu** [comment]
Hey, instead of creating a new PR, you can keep working on this current one to avoid cluttering the repo :)

@statelesshz or @jiaqiw09, based on your great work on previous Chinese translations, would you be interested in quickly reviewing? 🤗

**Comment by wwwbai** [comment]
> Hey, instead of creating a new PR, you can keep working on this current one to avoid cluttering the repo :)
> 
> @statelesshz or @jiaqiw09, based on your great work on previous Chinese translations, would you be interested in quickly reviewing? 🤗

Okay, I understand, thank you very much for your guidance. I have submitted the latest revision 'translated2'. Can it be reviewed now?

File: docs/source/zh/attention.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <img scale="50 %" align="center" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local_attention_mask.png"/>
+</div>
+
+使用参数更少的注意力矩阵，可以让模型处理更长的输入序列。
+
+## 其他技巧
+
+### 轴向位置编码
+
+[Reformer](model_doc/reformer)模型使用轴向位置编码：在传统的transformer模型中，位置编码矩阵E的大小是\\(l\\)乘以\\(d\\)，其中\\(l\\)是序列长度，\\(d\\)是隐藏状态的维度。如果你有非常长的文本，这个矩阵可能会非常大，占用GPU的大量空间。为了缓解这个问题，轴向位置编码将这个大矩阵E分解成两个较小的矩阵E1和E2，它们的维度分别是\\(l_{1} \times d_{1}\\) 和\\(l_{2} \times d_{2}\\)，满足\\(l_{1} \times l_{2} = l\\)和\\(d_{1} + d_{2} = d\\)（通过长度的乘积，最终得到的矩阵要小得多）。在E中，对于时间步\\(j\\) 的嵌入是通过连接E1中时间步 \\(j \% l1\\) 的嵌入和E2中时间步\\(j // l1\\)的嵌入来获得的。
```
Comment (statelesshz) [inline docs/source/zh/attention.md:36]:
```suggestion
[Reformer](model_doc/reformer)模型使用轴向位置编码：在传统的transformer模型中，位置编码矩阵E的大小是\\(l\\)乘以\\(d\\)，其中\\(l\\)是序列长度，\\(d\\)是隐藏状态的维度。如果你有非常长的文本，这个矩阵可能会非常大，将会占用大量的GPU显存。为了缓解这个问题，轴向位置编码将这个大矩阵E分解成两个较小的矩阵E1和E2，它们的维度分别是\\(l_{1} \times d_{1}\\) 和\\(l_{2} \times d_{2}\\)，满足\\(l_{1} \times l_{2} = l\\)和\\(d_{1} + d_{2} = d\\)（通过长度的乘积，最终得到的矩阵要小得多）。在E中，对于时间步\\(j\\) 的嵌入是通过连接E1中时间步 \\(j \% l1\\) 的嵌入和E2中时间步\\(j // l1\\)的嵌入来获得的。
```

**Review by ji-huazhong — COMMENTED** [review]
Only a small nit

**Comment by wwwbai** [comment]
> Only a small nit

Thank you very much for your valuable suggestions. I have submitted the revised changes accordingly.

**Review by stevhliu — APPROVED** [review]
LGTM and thanks for the review @statelesshz!

**Pull request merged**